### PR TITLE
#200: Added all details to the multiple-record-view

### PR DESCRIPTION
**Pull request by oskarhinc**

### CODE REVIEW

File: src/main/webapp/app/modules/conflicts/multiple/multiple-record-view.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export class MultipleRecordView extends React.Component<IMultipleRecordViewProp,
               <h4 className="from">
                 <Translate contentKey="multiRecordView.yourData" />
               </h4>
-              <Details organization={baseRecord.record.organization} {...this.props} exclusions={baseRecord.record.exclusions} />
+              <Details activity={baseRecord} {...this.props} exclusions={baseRecord.record.exclusions} isBaseRecord />
```
Comment (wojtekbu) [inline src/main/webapp/app/modules/conflicts/multiple/multiple-record-view.tsx]: Why 'isBaseRecord' default value is set to true? There will be always more children.

File: src/main/webapp/app/modules/conflicts/single/details.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    const sideSection = firstMatch ? (
+      <Col sm="3">
+        <Jumbotron className="jumbotron">
+          <div className="jumbotron-header">
+            <h4 className="jumbotron-header-title">
+              <Translate contentKey="singleRecordView.details.compare.title" />
+            </h4>
+            <p className="jumbotron-header-text">
+              {activity.organizationMatches.length}
+              {activity.organizationMatches.length === 1 ? (
```
Comment (wojtekbu) [inline src/main/webapp/app/modules/conflicts/single/details.tsx]: Why do you need this upper {activity.organizationMatches.length} ?

File: src/main/webapp/app/modules/conflicts/single/details.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                  ? `${firstMatch.partnerVersionName}`
+                  : `${firstMatch.partnerVersionName} +${activity.organizationMatches.length - 1}`}
+              </Link>
+            </div>
+          </div>
+        </Jumbotron>
+      </Col>
+    ) : null;
+
+    const columnSize = 6;
```
Comment (wojtekbu) [inline src/main/webapp/app/modules/conflicts/single/details.tsx]: please make it UPPERCASE

File: src/main/webapp/app/modules/conflicts/multiple/multiple-record-view.tsx (hunk 1 above)
Comment (oskarhinc) [reply src/main/webapp/app/modules/conflicts/multiple/multiple-record-view.tsx]: It's for the multi-record-view to indicate how some elements should be visible

File: src/main/webapp/app/modules/conflicts/single/details.tsx (hunk 3 above)
Comment (oskarhinc) [reply src/main/webapp/app/modules/conflicts/single/details.tsx]: It's not global const value but rather a local one, but it won't be changed so tsLisnt suggests making it 'const'

File: src/main/webapp/app/modules/conflicts/single/details.tsx (hunk 2 above)
Comment (oskarhinc) [reply src/main/webapp/app/modules/conflicts/single/details.tsx]: To indicate the number of partners that have similar records

**Pull request merged**